common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -252,6 +260,7 @@ def student_view(self, context):
             'next_url': context.get('next_url'),
             'prev_url': context.get('prev_url'),
             'override_hidden_exam': masquerading and special_exam_html is not None,
+            'gated_banner': content_milestones and self.runtime.user_is_staff,
```

Comment:
I'm still confused by the logic here - shouldn't it be `and not self.runtime.user_is_staff`? If the user is staff, the gated banner should not be shown, correct?